.github/workflows/ci-pkg-install.yml

```diff
@@ -6,6 +6,14 @@ on:
     branches: [master, "release/*"]
   pull_request:
     branches: [master, "release/*"]
+    paths:
```

Comment:
We cannot add paths here because these jobs are required for all PRs

Reply:
So let's add them to group-check?

Reply:
This is probably not necessary if you are going to trim the job in #15105. I would merge this PR without this addition

Reply:
@carmocca You wanted to drop nb jobs, right, so this is one way...

Reply:
Okay. Do you want me to push the change? It would be better to leave it for a separate PR as it will require that an admin changes the list of required jobs.

Reply:
yes, lest puch the addition to group-check :+1: